app/serializers/topic_tags_mixin.rb

```diff
@@ -0,0 +1,13 @@
+module TopicTagsMixin
```

Comment:
I would move this into a `concerns` folder under `serializers` so that it is clear this is a mixin.

Reply:
Nice idea to use a mixin though :+1: We get to nuke so many lines of code.